src/utility/deprecated.jl

```diff
@@ -0,0 +1,111 @@
+"""
+==========
+DEPRECATED 
+==========
+"""
+function criteria_data_map()
```

Comment:
I assume this is a top-level comment and not a deprecation of the function.

```suggestion
"""

function criteria_data_map()
```

Otherwise, use the `@deprecate` macro (https://invenia.github.io/blog/2022/06/17/deprecating-in-julia/)